### DISCUSSION TITLE
<title> 404 error while publishing a html document using index.html but still showing error

### DISCUSSION THREAD

**Discussion by govardhanreddy001** [community/community#60968] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Screenshot (16)](https://github.com/community/community/assets/126277387/2f168d4b-5ce4-4883-bc18-1bb959796cb5)

**Comment by radar-blip** [comment]
Some of your links in the index.html are wrong,

Try changing this:
`<h3><a href="../2.4 Movie Ranking Project/index.html"> Movie Ranking Project </a></h3>`

to this:
`<h3><a href="movie-project.html"> Movie Ranking Project </a></h3>`

And this:
`<h3><a href="./public/birthday-invite.html">Birthday Invite Project </a></h3>`

to this:
`<h3><a href="birthday-invite.html">Birthday Invite Project </a></h3>`

**Reply by govardhanreddy001** [reply]
still facing error site not publishing

**Reply by govardhanreddy001** [reply]
![Screenshot (18)](https://github.com/community/community/assets/126277387/7e2a4441-33d2-4df7-a768-f57bae6ec2ec)
